.github/workflows/docker.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: ./contrib/docker/
-          file: {context}/Dockerfile.root
+          file: Dockerfile.root
```

Comment:
If that still doesn't work, maybe one of the following does
- `"{{context}}/Dockerfile.root"`
- `./contrib/docker/Dockerfile.root`